avm/res/document-db/database-account/main.bicep

```diff
@@ -106,7 +109,13 @@ param roleAssignments roleAssignmentType[]?
 param dataPlaneRoleDefinitions dataPlaneRoleDefinitionType[]?
```

Comment:
Related to the update to `dataPlaneRoleAssignments`, let's please rename this one to `sqlRoleDefinitions: sqlRoleDefinitionType` while we're at it